app/config/main.py

```diff
@@ -40,5 +40,6 @@ def load_config(config_dir: Path = None) -> Config:
         tg_client=TgClientConfig(bot_token=_bot_token),
         storage=load_storage(config_file_data["storage"]),
         report_karma_award=config_file_data.get("report_karma_award", 0),
+        time_to_remove_report_karma_award_messages=config_file_data.get("time_to_remove_report_karma_award_messages", 300),
```

Comment:
Let's call it `report_award_cleanup_delay` and set default 1 hour:
```suggestion
        report_award_cleanup_delay=config_file_data.get("report_award_cleanup_delay", 3600),
```